libs/database/src/lib/aggregations/get-users-relation.aggregation.ts

```diff
@@ -198,3 +208,60 @@ export const pipelineOfUserRelationFollowingCount = (
     ],
   ];
 };
+
+export const pipelineOfUserRelationFollowersV2 = (
+  params: GetUserRelationParamsV2,
+) => {
+  return [...userFollowQueryV2(params), ...[{ $limit: params.limit }]];
+};
+
+export const pipelineOfUserRelationFollowersCountV2 = (
+  params: GetUserRelationParamsV2,
+) => {
+  return [
+    ...userFollowQueryV2(params),
+    ...[
+      {
+        $count: 'total',
+      },
+    ],
+  ];
+};
+
+const filterTypeV2 = (userType?: string[]) => {
+  if (userType)
+    return {
+      'user_relation.type': { $in: userType },
+    };
+};
+
+const userFollowQueryV2 = (params: GetUserRelationParamsV2) => {
+  return [
+    {
+      $lookup: {
+        from: 'users',
+        localField: 'user',
+        foreignField: '_id',
+        as: 'user_relation',
```

Comment:
```suggestion
        as: 'user',
```
Replace it with `user`, not necessary to create new property